tests_config/run_spec.js

```diff
@@ -39,7 +39,6 @@ const unstableTests = new Map(
       (options) => options.proseWrap === "always",
     ],
     ["js/no-semi/comments.js", (options) => options.semi === false],
-    "yaml/prettier-ignore/document.yml",
```

Comment:
This is test

Reply:
It's unstable before, so it's listed here.

Reply:
Sorry I didn't noticed